app/components/result_summary_component.rb

```diff
@@ -0,0 +1,55 @@
+class ResultSummaryComponent < ViewComponent::Base
+  include ViewHelper
+
+  attr_reader :results
+
+  def initialize(results:)
+    @results = results
+  end
+
+  def call
+    [
+      course_found_text,
+      subject_names,
+      with_location_text,
+      with_england_text,
+      with_provider_text,
+    ].compact.join(' ')
+  end
+
+private
+
+  def subject_names
+    results.subjects.map { |subject| subject[:subject_name].downcase }.to_sentence(last_word_connector: ' and ')
```

Comment:
Proper nouns needs correct casing

Reply:
There's a separate ticket for this (which Simon knows about). We can cross port the approach implemented and handle this bit too. I'll add it to that ticket